api/src/main/java/javax/el/ELUtil.java

```diff
@@ -222,13 +222,22 @@ static Method findMethod(Class<?> klass, String methodName, Class<?>[] paramType
      */
     static Object invokeMethod(ELContext context, Method method, Object base, Object[] params) {
 
-        Object[] parameters = buildParameters(context, method.getParameterTypes(), method.isVarArgs(), params);
         try {
+            int paramCount = params == null ? 0 : params.length;
+            if (paramCount == 0) {
+                for (Class ParameterType : method.getParameterTypes()) {
+                    if (ParameterType.getName().equals("javax.faces.event.AjaxBehaviorEvent")) {
```

Comment:
I don't like it much, relying on class names is extremely fragile.
Also ... what it does ... 
IF we received null or zero parameters THEN look at the method signature, and IF ANY of those parameters is of exact class name execute the method with what we received.

IN ALL OTHER cases transform parameters to new parameter array and execute the method.